### PR TITLE
 Convert unnecessary storage item to static.

### PR DESCRIPTION
**Pull request by gavofyork**
There was no need to keep minimum block time as a storage item. This just switches it to a static Getter.

### CODE REVIEW

File: srml/babe/src/lib.rs [hunk 1]
```diff
@@ -193,7 +193,7 @@ impl<T: Trait> Module<T> {
 	pub fn slot_duration() -> T::Moment {
 		// we double the minimum block-period so each author can always propose within
 		// the majority of their slot.
-		<timestamp::Module<T>>::minimum_period().saturating_mul(2.into())
+		<T as timestamp::Trait>::MinimumPeriod::get().saturating_mul(2.into())
```
Comment (andresilva) [inline srml/babe/src/lib.rs]: Missing import of `srml_support::traits::Get` in this file I think

Comment (shawntabrizi) [reply srml/babe/src/lib.rs]: I patched this real quick

File: srml/timestamp/src/lib.rs [hunk 2]
```diff
@@ -208,23 +210,30 @@ pub trait Trait: system::Trait {
 
 	/// Something which can be notified when the timestamp is set. Set this to `()` if not needed.
 	type OnTimestampSet: OnTimestampSet<Self::Moment>;
+
+	/// The minimum period between blocks. Beware that this is different to the *expected* period
+	/// that the block production apparatus provides. Your chosen consensus system will generally
+	/// work with this to determine a sensible block time. e.g. For Aura, it will be double this
+	/// period on default settings.
+	type MinimumPeriod: Get<Self::Moment>;
```
Comment (bkchr) [inline srml/timestamp/src/lib.rs]: We still need to export this value in `decl_module!`.

**Comment by gavofyork** [comment]
looks good.

**Comment by andresilva** [comment]
Integration test failed due to some unrelated docker issue, restarted.

**Pull request merged**